src/Worker/GrpcWorkerInterface.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Baldinof\RoadRunnerBundle\Worker;
```

Comment:
If we remove an interface, that would mean a 3.0 release. Maybe you can just make the `GrpcWorkerInterface` extends the `WorkerInterface`, and deprecate `GrpcWorkerInterface`.

So we keep backward compatibility.

Reply:
added deprecation